service/topology_coordinator.cc

```diff
@@ -1173,7 +1173,10 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                 case locator::tablet_transition_stage::streaming:
                     if (drain) {
                         utils::get_local_injector().inject("stream_tablet_fail_on_drain",
-                                        [] { throw std::runtime_error("stream_tablet failed due to error injection"); });
+                                        [] {
+                                            rtlogger.info("Injecting stream_tablet_fail_on_drain");
+                                            throw std::runtime_error("stream_tablet failed due to error injection");
```

Comment:
Is this some left over from a test you wrote but didn't push?

Reply:
The test is preexisting.
It's important to log that an error is being injected to be able to match the node log with the test log.